src/s3/commands/downloadFileAs.ts

```diff
@@ -149,7 +157,7 @@ async function promptForSaveLocation(fileName: string, window: Window): Promise<
         ? { [`*${extension}`]: [extension.slice(1)], 'All Files': ['*'] }
         : { 'All Files': ['*'] }
 
-    const downloadPath = path.join(downloadsDir(), fileName)
+    const downloadPath = path.join(saveLocation, fileName)
     return window.showSaveDialog({
         defaultUri: vscode.Uri.file(downloadPath),
         saveLabel: localize('AWS.s3.downloadFile.saveButton', 'Download'),
```

Comment:
What happens if we do this?
```suggestion
    return window.showSaveDialog({
        saveLabel: localize('AWS.s3.downloadFile.saveButton', 'Download'),
```

Reply:
This will pull VSC's 'last used download path', but there's two problems.

1. This doesn't add the file name of what's being downloaded to the save window. Meaning the user would need to manually enter a name every time.
2. This will NOT update VSC's 'last used download path'.